lib/iiif_manifest/v3/manifest_builder.rb

```diff
@@ -29,8 +29,32 @@ def to_h
         @to_h ||= builders.new(work).apply(top_record)
       end
 
+      # Utility method to wrap the obj into a IIIF V3 compliant language map as needed.
+      def self.language_map(obj)
+        # self.valid_language_map?(obj) ? obj : self.transform(obj)
+        obj.is_a?(Hash) ? transform_hash_value(obj) : transform_obj_value(obj)
+      end
+
+      private_class_method
```

Comment:
I see the `class << self` pattern more.  Do you mind using it?  
Number 2 is an example of this pattern: http://codebeerstartups.com/2016/07/private-class-methods-in-ruby/  I usually see it at the beginning of the class before anything else like the `attr_reader` or `initialize`.